frontend/src/components/auth/Login.tsx

```diff
@@ -21,6 +23,8 @@ type GoogleErrorResponse = {
 
 const Login = (): React.ReactElement => {
   const { authenticatedUser, setAuthenticatedUser } = useContext(AuthContext);
+  const [loginError, setLoginError] = useState<PresentableError>();
```

Comment:
Not the purpose of this PR but I think when we eventually code the error handling on frontend it should be one state variable.

errors = {
loginError : True
emailError: False
}

Reply:
Ah true, I thought it might be easier to separate the different "areas" of errors - loginError is a box on top of the whole thing, emailError is a piece of text under the email input, and later passwordError will be the text under password.
What disadvantages are there to having separate error state variables?
I personally find state objects are a bit more annoying to work with than separate state variables, but that might just be me :D

Reply:
I agree, it's pretty much just preference.

The only two small advantages are less declarations (cleaner code ig) and the other is let's say you wanted to disable a button ("sign up" for example) if there are any errors.

If you modify the object logic to only add the field if you encounter an error, you'll end up with something like this.

```
error = {
loginError = True
}
<button style={{disabled: error.length != 0 }} />
// instead of
<button style={{disabled: !loginError || !emailError ... (possibly much more) }} />
```

super minor stuff tho, either works.

Reply:
aha great point on the disabling a button example! seems like this decision will need to be made on a case-by-case basis, depending on what errors and behaviour we need to support.
we should definitely keep this in mind for code reviews when we get to error handling for the login page (and future user input components) :)